fillager/src/test/kotlin/no/nav/soknad/arkivering/soknadsfillager/rest/GetFilesTests.kt

```diff
@@ -65,6 +65,22 @@ class GetFilesTests {
 		assertTrue(nonExistentFiles.all { it.status == statusNotFound })
 	}
 
+	@Test
+	fun `Get files - one id do not exist`() {
```

Comment:
Vurder å legge til tilsvarende test for getFilesMetadata

Reply:
Lagt til